src/Microsoft.Dnx.Tooling/Restore/RestoreCommand.cs

```diff
@@ -400,11 +400,14 @@ public RestoreCommand(FrameworkName fallbackFramework)
                         {
                             Reports.Verbose.WriteLine($"Restoring packages for {pair.context.FrameworkName} on {runtimeName}...");
                             var runtimeSpecs = new List<RuntimeSpec>();
+                            var runtimesw = Stopwatch.StartNew();
                             FindRuntimeSpecs(
                                 runtimeName,
                                 runtimeFiles,
                                 runtimeSpecs,
                                 _ => false);
+                            runtimesw.Stop();
+                            Reports.Verbose.WriteLine($" Scanned Runtime graph in {runtimesw.ElapsedMilliseconds:0.00}ms");
```

Comment:
This is mostly debug spew but I'll leave it because it's verbose and I plan to go through and clean some of this up